cli/src/lib.rs

```diff
@@ -62,23 +65,19 @@ impl Default for Config
     }
 }
 
-pub fn load_config(path: &Path) -> Result<Config>
-{
-    let file = std::fs::File::open(path)
-        .context("Error loading config. Please run `uno init` first.")?;
-    let config: Config = ron::de::from_reader(file)
-        .context("Error parsing config.")?;
+pub fn load_config(path: &Path) -> Result<Config> {
```

Comment:
looks like we can configure 
```
brace_style = AlwaysNextLine
```
 I'll do a rustfmt config pass during cooldown

Reply:
actually, if we did that now it would reduce the formatting thrashing on this pr

Reply:
I don't really care though not the end of the world to have these reformat in a few weeks.

Reply:
actually I'll do it tonight it's not that bad